### PR TITLE
PyPI how-to

### PR DESCRIPTION
**Pull request by chosak**
This commit includes a draft how-to document for publishing CFPB Python
packages to PyPI.

## Additions

- New `pypi.md`.

## Changes

- Link to `pypi.md` in the `README`.

## Review

- @rosskarchner @willbarton @ascott1 

## Checklist

* [X] Changes are limited to a single goal (no scope creep)
* [X] Code can be automatically merged (no conflicts)

### CODE REVIEW

File: pypi.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+A simple set of commands to accomplish this looks like:
+
+1. `python setup.py sdist bdist_wheel` (creates both a source distribution and a [Wheel archive](https://pip.pypa.io/en/stable/reference/pip_wheel/) of your project, and places them in a `dist` subdirectory.
+2. `twine upload dist/*.whl` (pushes your new distribution files to PyPI)
+
+The contents of your project's `setup.py` file determine how your code is packaged and how it gets published on PyPI. There are a number of useful and important fields that you should include in this file:
+
+- `url`: link to GitHub repository.
+- `author`: should be CFPB.
+- `author_email`: should be tech@cfpb.gov.
```
Comment (ascott1) [inline pypi.md]: Did we decide on `tech@cfpb.gov` over the `devops@cfpb.gov` address?

Comment (chosak) [reply pypi.md]: I left this as `tech@` to keep consistency with currently published projects.

Comment (ascott1) [reply pypi.md]: I spoke with @chosak offline. We decided to move forward with `tech@cfpb.gov` as recommended here.

**Comment by chosak** [comment]
@Scotchester nice catch; I will defer to @ascott1 on this one.

Looking into this some more, it appears that an alternate, and perhaps better, way to specify the license is via the trove classifiers, which support the list prefixed with "License" [here](https://pypi.python.org/pypi?%3Aaction=list_classifiers). I suppose we could include both of these. I don't know whether the `setup.py` `license` field even supports multiple options, and [the docs say](https://docs.python.org/3.6/distutils/setupscript.html#additional-meta-data)

> The license field is a text indicating the license covering the package where the license is not a selection from the “License” Trove classifiers. See the Classifier field. Notice that there’s a licence distribution option which is deprecated but still acts as an alias for license.

So this implies that perhaps we should be using the classifiers instead.

**Comment by ascott1** [comment]
In our npm packages we specify the CC0 License, since there isn't a [SPDX Public Domain License](https://spdx.org/licenses/). If classifiers work, that sounds great, but otherwise we can leave CC0 and request that the license distinction be placed in the README and LICENSE files.

**Comment by Scotchester** [comment]
We should be following the advice of [the package.json spec](https://docs.npmjs.com/files/package.json#license) for npm packages:

> If you are using a license that hasn't been assigned an SPDX identifier, or if you are using a custom license, use a string value like this one:
>
> `{ "license" : "SEE LICENSE IN <filename>" }`
>
> Then include a file named `<filename>` at the top level of the package.

We've started doing this [in a few places](https://github.com/search?utf8=%E2%9C%93&q=org%3Acfpb+%22see+license+in%22&type=Code), but haven't propagated it far and wide yet.

**Comment by Scotchester** [comment]
PR opened to update the `package.json` template in our npm package generator, which I will use as a reference when editing the authoring guidance in this repo.

**Comment by chosak** [comment]
I've made a slight change to the licensing guidelines in 44cacb9 based on various pieces of feedback. @Scotchester @ascott1 would you mind taking another look?

**Pull request merged**